packages/sources/dxfeed/src/config.ts

```diff
@@ -5,10 +5,13 @@ export const NAME = 'DXFEED'
 
 export const DEFAULT_ENDPOINT = 'price'
 export const DEMO_ENDPOINT = 'https://tools.dxfeed.com/webservice/rest'
+export const DEFAULT_WS_ENDPOINT =
+  'wss://chainlink:INei2ookaexi3m@tools.dxfeed.com/chainlink/cometd'
```

Comment:
Is this for public use?

Reply:
ah good catch.  No it isn't.  I forgot to remove it.